schema.graphql

```diff
@@ -0,0 +1,54 @@
+
+type Query {
+  frameworks: [Framework!]
+  framework(id: ID!): Framework
+
+  languages: [Language!]
+  language(id: ID!): Language
+
+  concurrencys: [Concurrency!]
+  concurrency(id: ID!): Concurrency
+
+  keys: [Key!]
+  key(id: ID!): Key
+
+  metrics: [Metric!]
+  metric(id: ID!): Metric
+
+  values: [Value!]
+  value(id: ID!): Value
+}
+
+type Framework {
+  id: ID
+  label: String!
+  language: Language
+}
+
+type Langauge {
+  id: ID
+  label: String!
+}
+
+type Concurrency {
+  id: ID
+  level: Int
+}
+
+type Key {
+  id: ID
+  label: String!
+}
+
+type Metric {
+  id: ID
+  framework: Framework
+  value: Value
+  concurrency: Concurrency
+}
+
+type Value {
+  id: ID
+  value: Int
```

Comment:
@ohler55 Those are values stored in database stored as `float`
https://github.com/the-benchmarker/benchmarker-data/blob/dd0549b4798df1b37cda08124c468e86c43c6162/db/migrations/0005_values.rb#L6

Reply:
I would have thought so too. The seed script seemed to use setting an attribute called value with an int. Maybe that is meant to be something else. I did not see a float being set in the seeding though.

Reply:
If you know what attributes are in the Postgres tables I'll update based on that. That would be better than inspecting a seeded object after it is created.

Reply:
Absolutely. In fact, there are stored as `double precision` in **postgresql** (honestly in prefer not to take care of other engines) and considered as `Float` in `Ruby`

Reply:
No matter if seeds uses `Integer` :stuck_out_tongue: they are automatically **casted** before storing in `database`

Reply:
Change pushed.